capsule/src/main/java/Capsule.java

```diff
@@ -2470,6 +2454,65 @@ private void resolveNativeDependencies() {
         return new ArrayList<String>(jvmArgs.values());
     }
 
+    private List<String> parseCommandLineArguments(String toProcess) {
```

Comment:
How do you want me to state that this is heavily inspired by http://grepcode.com/file/repo1.maven.org/maven2/org.apache.ant/ant/1.9.5/org/apache/tools/ant/types/Commandline.java#Commandline.translateCommandline%28java.lang.String%29 ?